tests/net/socket/tls/src/main.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(net_test, CONFIG_NET_SOCKETS_LOG_LEVEL);
+
+#include <ztest_assert.h>
+#include <fcntl.h>
+#include <net/socket.h>
+#include <net/tls_credentials.h>
+
+#include "../../socket_helpers.h"
+
+#define TEST_STR_SMALL "test"
```

Comment:
This define is not used anywhere. Same goes with most of the other defines below.

Reply:
Thanks for noticing. I've removed unused defines.